app/components/product-destroy/component.js

```diff
@@ -0,0 +1,19 @@
+import Ember from 'ember';
+
+const { Component } = Ember;
+
+export default Component.extend({
+  actions: {
+    launchConfirmDialog() {
```

Comment:
you can simplify `launchConfirmDialog` and `cancelConfirm` with one toggle furnction i.e.:

```` js
togggle() {
  this.toggleProperty('confirmShow');
}
````

https://www.emberjs.com/api/ember/2.14/classes/Ember.Observable/methods/toggleProperty?anchor=toggleProperty